.github/workflows/sync-from-upstream.yaml

```diff
@@ -0,0 +1,39 @@
+name: Sync from upstream
+
+on:
+  schedule:
+    - cron: '0 7 */7 * *' # At 07:00 on every 7th day-of-month
+
+  # Allows to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  sync-app-repository:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Sync falco from upstream
+        uses: giantswarm/subtree-util@v0.1.2
+        with:
```

Comment:
If this would be a repository with a single chart in it, I would suggest using the `source_tag_wildcard`, but! I think currently this breaks this repository. so leaving this out for now is okay IMO